BaseClasses.py

```diff
@@ -516,6 +516,49 @@ def get_spheres(self) -> Iterator[Set[Location]]:
                 state.collect(location.item, True, location)
             locations -= sphere
 
+    def get_sendable_spheres(self) -> Iterator[Set[Location]]:
+        """
+        yields a set of multiserver sendable locations (location.item.code: int) for each logical sphere
+
+        If there are unreachable locations, the last sphere of reachable locations is followed by an empty set,
+        and then a set of all of the unreachable locations.
+        """
+        state = CollectionState(self)
+        locations: Set[Location] = set()
+        events: Set[Location] = set()
+        for location in self.get_filled_locations():
+            if type(location.item.code) is int:
```

Comment:
Is there a reason this doesn't use isinstance?

Reply:
overkill, there is no reason it should ever be a subclass, so it's wasted CPU cycles to check.

Reply:
Wait, but is this actually faster than `is not None` as well?

Reply:
Anyway not a big deal just curious why this was chosen

Reply:
it can be a tuple as well.

Reply:
It CAN???